admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/pathbrowser/template.vue

```diff
@@ -547,8 +551,11 @@ export default {
         'image/svg+xml'].indexOf(item.mimeType) >= 0
     },
     isImageExtension(item) {
-
-      return item.path.match(/.(jpg|jpeg|png|gif|svg)$/i)
+      if (item.path) {
```

Comment:
this is the major change here

Reply:
Sure. That answers my question. However, we probably should not change the formatting in future PRs as this can lead to very complicated merge conflicts as well as make it difficult for other developers to quickly determine the change. In my opinion, if we want to reformat our code base to follow a well-defined formatting guideline for the project, we should do this a separate refactoring ticket and PR. I'll merge this later today. Thanks again!